Makefile

```diff
@@ -81,10 +83,20 @@ check-license: ## check license headers in source files
 	@./.tool/check-license
 
 .PHONY: lint
-lint: .install.lint ## lint check of Go files using golangci-lint
-	@echo "checking lint"
+
+.PHONY: lint
+lint: lint-go lint-md ## Run all linters
+
+.PHONY: lint-go
+lint-go: .install.lint ## lint check of Go files using golangci-lint
+	@echo "checking Go lint"
 	@GO111MODULE=on $(GOPATH)/bin/golangci-lint run
 
+.PHONY: lint-md
+lint-md: ## Run linting for markdown
+	docker run --rm -v "$(PWD):/workdir:ro" davidanson/markdownlint-cli2:$(MARKDOWN_LINT_VER) \
```

Comment:
i know it is implied, but can we add the `docker.io/` transport to the container name

Reply:
Done!